### PR TITLE
first commit for #182; Bmsy_for_H4010_top

### PR DESCRIPTION
**Pull request by Rick-Methot-NOAA**
## What issue(s) does this PR address? Describe and add issue numbers, if applicable.

Link issue(s) here: #182 

## What tests have been done? Upload any model input files created for testing in a zip file, if possible.
verified with simple example
## What tests/review still need to be done? Who can do it, and by when is it needed (ideally)?
none, but checking variance would be good idea
## Check which is true. This PR requires:

- [ ] no further changes to r4ss
- [ ] no further changes to the manual
- [ ] no further changes to SSI (the SS3 GUI)
- [ ] no further changes to the stock synthesis change log (new features, bug reports)

### CODE REVIEW

**Review by kellijohnson-NOAA — COMMENTED** [review]
I was able to compile this branch locally and I too ran the simple example as practice of running through the process. After running simple and looking at some of the comments and warning messages I think with just a little bit more effort we could increase consistency to help future users. For example "control rule bottom" should be "control rule cutoff" in all instances.

I am not sure what is standard practice here? Does the review commit the proposed changes in this branch or a fork or would @Rick-Methot-NOAA be responsible for making changes if he agrees with the comments?

**Comment by Rick-Methot-NOAA** [comment]
consistency is good and not my forte.  I am fine with use of control rule throughout and glad if someone else can make those edits.

change to the .bat file in this commit was accidental and harmless.  It need not be isolated in its own commit.

File: SS_benchfore.tpl [hunk 1]
```diff
@@ -1989,6 +1989,16 @@ FUNCTION void Get_Forecast()
         report5 << endl;
       }
     }
+    
+    if (H4010_top_rd < 0.0)
```
[the remark this answers is not in the document]
Comment (k-doering-NOAA) [reply SS_benchfore.tpl:1993]: yes, space after (if () ... I usually forget this space, to be honest, but am slowly learning to put it in (styler for r code adds this space, also)!

**Review by kellijohnson-NOAA — APPROVED** [review]
[4a19422](https://github.com/nmfs-stock-synthesis/stock-synthesis/pull/322/commits/4a194221dcc92d8753736930d67d93d8fab9c55b) increases consistency in comments. I think this can be merged once checks show that they are all passing.

**Comment by kellijohnson-NOAA** [comment]
# Info for change log

## Version
3.30.20

## Date
2022-05-31

## Description
Option for control rule inflection point to use Bmsy / SSB_unf
instead of user-specified value. If read in value is -1 then,
Stock Synthesis will use calculated Bmsy/SSB_unf to override input value.

## Issue
#182

## Input change
No

## Action
revise

## Topic
forecast

## Type
input, calc

**Comment by Rick-Methot-NOAA** [comment]
Thanks Kelli for the improvements.

**Pull request merged**